dlite_plugins/singlefiledatanode.py

```diff
@@ -0,0 +1,64 @@
+"""
+Dlite plugin to convert between DLite instances or files and AiiDA single file datanode.
+To be included in ExecFlow.
+
+"""
+
+from __future__ import annotations
+
+import json
+
+import dlite
+from dlite.options import Options
+
+
+class singlefiledatanode(dlite.DLiteStorageBase):
+    """General description of the Python storage plugin."""
+
+    def open(self, location, options=None):
+        """Open storage.
+
+        Arguments:
+            location:
+            options: Additional options for this storage driver.
```

Comment:
Arguments could be better documented. 
Especially the `driver` option would be good to have documented.